### DISCUSSION TITLE
Is it possible to make multiple text inputs?

### DISCUSSION THREAD

**Discussion by MikaelFangel** [charmbracelet/gum#288] [Q&A]
Is it possible to take multiple text inputs simultaneously just as with bubbletea? If yes then how?

The desired result is this bubbletea example:
[https://github.com/charmbracelet/bubbletea/tree/master/examples/textinputs](https://github.com/charmbracelet/bubbletea/tree/master/examples/textinputs)

**Comment by beltagyy** [comment]
Gum does not provide a built-in multi-input component - each gum input call is a standalone screen.

To get multiple inputs on one screen, you have two options:
1. Chain gum input calls in your shell script and store results in variables:
   name=$(gum input --placeholder "Name")
   email=$(gum input --placeholder "Email")

2. Use Bubble Tea directly. Gum is built on Bubble Tea, so you can copy the textinputs example from the Bubble Tea repo into a small Go program for the exact UI you want.

If your use case is a form, consider using a small Bubble Tea app or the hummus/bubbles form component rather than trying to force multiple standalone gum inputs into one view.

**Reply by MikaelFangel** [reply]
Fair and thank you for your response but to be honest I don't remember the use case I had 4 years ago 😅
